contracts-abi/contracts/hts-precompile/IHederaAccountService.sol/IHederaAccountService.json

```diff
@@ -0,0 +1,60 @@
+[
```

Comment:
Is this file needed? If yes, is it needed here? I don't see where is it used anywhere in the PR.

Reply:
Ah good catch but yes it's the artifact for the IHederaAccountService interface Luke forgot to check in.